tests/Unit/Domain/FunctionsOfTime/Test_SettleToConstant.cpp

```diff
@@ -13,7 +13,7 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 namespace {
-void test(const domain::FunctionsOfTime::FunctionOfTime& f_of_t,
+void test(const std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime> f_of_t,
```

Comment:
Should this argument be passed by reference? I worry you are copying an owning unique ptr.

Reply:
This is done after bringing in #1856 :+1: